bootstrap/eks/internal/userdata/node.go

```diff
@@ -20,18 +20,37 @@ import (
 	"bytes"
 	"fmt"
 	"text/template"
+
+	"github.com/alessio/shellescape"
 )
 
 const (
 	nodeUserData = `#!/bin/bash
-/etc/eks/bootstrap.sh {{.ClusterName}} {{- template "args" .KubeletExtraArgs }}
+/etc/eks/bootstrap.sh {{.ClusterName}} {{- template "args" . }}
 `
 )
 
 // NodeInput defines the context to generate a node user data.
 type NodeInput struct {
-	ClusterName      string
-	KubeletExtraArgs map[string]string
+	ClusterName           string
+	KubeletExtraArgs      map[string]string
+	ContainerRuntime      *string
+	DNSClusterIP          *string
+	DockerConfigJSON      *string
+	APIRetryAttempts      *int
+	PauseContainerAccount *string
+	PauseContainerVersion *string
+	UseMaxPods            *bool
+	IPFamily              *string
+	ServiceIPV6Cidr       *string
```

Comment:
It'd be good to have a note here for both ServiceIPV6Cidr and IPFamily as this is not implemented yet and no-op.

Reply:
Added a comment